src/net_processing.cpp

```diff
@@ -3280,7 +3280,7 @@ bool PeerLogicValidation::ProcessMessages(CNode* pfrom, std::atomic<bool>& inter
             return false;
         // Just take one message
         msgs.splice(msgs.begin(), pfrom->vProcessMsg, pfrom->vProcessMsg.begin());
-        pfrom->nProcessQueueSize -= msgs.front().m_recv.size() + CMessageHeader::HEADER_SIZE;
+        pfrom->nProcessQueueSize -= msgs.front().m_raw_message_size;
```

Comment:
Had to do a double-take here... But this shouldn't change behaviour as `msgs.front().m_recv.size()` is always equal to `msg.m_message_size` because the `msgs.front().m_recv` hasn't advanced its `nReadPos` yet. Verified by placing an assert and running the unit and functional tests.